src/main/java/com/fasterxml/jackson/databind/jsontype/impl/AsDeductionTypeDeserializer.java

```diff
@@ -35,6 +35,8 @@
     // Bitmap of available fields in each subtype (including its parents)
     private final Map<BitSet, String> subtypeFingerprints;
 
+    private static final String EMPTY_CLASS_MARKER = "";
```

Comment:
Needs Javadoc to explain use.

Reply:
One comment: empty String is actually valid property name in JSON -- is this problematic?